Attributes.rb

```diff
@@ -0,0 +1,35 @@
+class Class
+
+	def my_attr_accessor(*args)
+		my_attr_reader(*args)
+		my_attr_writer(*args)
+	end
+
+	def my_attr_reader(*args)
+		args.each do |arg|
+		  self.class_eval("def #{arg};@#{arg};end")
+	    end
+	end
+
+	def my_attr_writer(*args)
+		args.each do |arg|
+		  self.class_eval("def #{arg}=(val);@#{arg}=val;end")
+	    end
+	end
+end
+
+
+
+class Person
+	# my_attr_reader :name,:age
+	# my_attr_writer :name,:age
+	my_attr_accessor :name,:age
+end
+
+person= Person.new
+
+person.name ="Soumya"
+person.age = 5
+
+puts person.name
+puts person.age
```

Comment:
Check with others.  they are doing perfectly.

-Manoj

-- 
Manoj P M
V2Solutions | The Next Level of Innovation
o: 408.694.3865 x436

On 24/08/15 10:13 am, sowjanyakunder wrote:
>
> In Attributes.rb 
> <https://github.com/sowjanyakunder/Person_details/pull/5#discussion_r37718985>:
>
> > +
> > +
> > +
> > +class Person
> > +	# my_attr_reader :name,:age
> > +	# my_attr_writer :name,:age
> > +	my_attr_accessor :name,:age
> > +end
> > +
> > +person= Person.new
> > +
> > +person.name ="Soumya"
> > +person.age = 5
> > +
> > +puts person.name
> > +puts person.age
>
> Sir,I am creating a new branch and then giving pull request.Is there 
> any other way that I can create different PR?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/sowjanyakunder/Person_details/pull/5/files#r37718985>.
>